style.css

```diff
@@ -1150,45 +1153,54 @@ button:active {
   }
 
   .popup-container {
-    display: flex;
-    flex-direction: row;
     position: fixed;
+    display: flex;
+    flex-direction: column;
     background-color: white;
     width: 100%;
     height: 100%;
     overflow-y: scroll;
   }
```

Comment:
- [ ] Great work here styling the popup, kindly add padding to your popup to match the design, below is a comparison
Desktop 
I have also highlighted in red the margin above the buttons that require to be minimized.

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/168318773-e6ede7a9-dab6-4dcb-aa3a-2405c043ca31.png) |  <img width="377" alt="image" src="https://user-images.githubusercontent.com/61408860/168319071-4fe82079-d559-4ee1-a158-73415d6d4abc.png">


Mobile
Kindly ensure the content in your design matches the design given.
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/168319328-6c092f63-5c6f-43e7-a2d8-cf8d7d91e785.png) |  ![image](https://user-images.githubusercontent.com/61408860/168319579-de16a224-f711-4070-a38a-ed3958d85c0e.png)